absl/debugging/internal/elf_mem_image.h

```diff
@@ -34,7 +34,7 @@
 #define ABSL_HAVE_ELF_MEM_IMAGE 1
 #endif
 
-#if ABSL_HAVE_ELF_MEM_IMAGE
+#if defined(ABSL_HAVE_ELF_MEM_IMAGE) && ABSL_HAVE_ELF_MEM_IMAGE
```

Comment:
`#if defined(ABSL_HAVE_ELF_MEM_IMAGE)` is good enough: the macros should either be 1 or undefined.

Reply:
Resolved by using #ifdef like above in the file.